### PR TITLE
[17.0][MIG] stock_lot_production_date: Migration to 17.0

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

**Comment by rolandojduartem** [comment]
LGTM :+1: 

cc @hugho-ad @Hernannc

**Review by rolandojduartem — APPROVED** [review]

**Comment by rolandojduartem** [comment]
Hi, @moylop260, please, could you merge it?

File: stock_lot_production_date/README.rst
Comment (moylop260) [inline stock_lot_production_date/README.rst]: Please, remove the contributor change but add as maintainer

I mean, add a input in the `__manifest__.py` file with:
`"maintainers": ["SodexisTeam"],`

Comment (rolandojduartem) [reply stock_lot_production_date/README.rst]: Also the change in the index.html

Comment (rolandojduartem) [reply stock_lot_production_date/README.rst]: @SodexisTeam

File: stock_lot_production_date/models/stock_lot.py
Comment (moylop260) [inline stock_lot_production_date/models/stock_lot.py]: Revert this change to original way, please

The original line is good for our guidelines

File: stock_lot_production_date/README.rst
Comment (rousseldenis) [reply stock_lot_production_date/README.rst]: @moylop260 This is not correct.

The philosophy behind "maintainers" is that should use individual github accounts, not organization ones.

**Comment by rousseldenis** [comment]
@sebalix Do you agree the maintainers change(adoption) ?

File: stock_lot_production_date/README.rst
Comment (SodexisTeam) [reply stock_lot_production_date/README.rst]: @rousseldenis We are going to follow your advice and start using individual account.

File: stock_lot_production_date/models/stock_lot.py
Comment (SodexisTeam) [reply stock_lot_production_date/models/stock_lot.py]: @moylop260 We have issue with pre-commit in this line so we used the "\", now we have removed that. Please check once again.

File: stock_lot_production_date/README.rst
Comment (rousseldenis) [reply stock_lot_production_date/README.rst]: @SodexisTeam yes, indeed, declaring a PR with organization account seems OK, but contributors and maintainers should rely on individual accounts. The company can appear on authors part when applicable.

Comment (moylop260) [reply stock_lot_production_date/README.rst]: > @rousseldenis This is not correct.

Thank you for your advice, I didn't know it

Could you refer your advice, please? (I didn't find any related with this "not correct")

> We are going to follow your advice and start using individual ACC

@SodexisTeam will you change it the value in maintainer?

File: stock_lot_production_date/models/stock_lot.py
Comment (moylop260) [reply stock_lot_production_date/models/stock_lot.py]: What issue?

I mean, the length of the line is good for the guidelines defined but not sure what was the problem?

File: stock_lot_production_date/README.rst
Comment (SodexisTeam) [reply stock_lot_production_date/README.rst]: @moylop260 We are working on it.

File: stock_lot_production_date/models/stock_lot.py
Comment (SodexisTeam) [reply stock_lot_production_date/models/stock_lot.py]: @moylop260 The error we got is 
ruff.....................................................................Failed
hook id: ruff
exit code: 1

stock_lot_production_date/models/stock_lot.py:11:89: E501 Line too long (94 > 88)
Found 1 error.

Actual length of the line in our file is 94 but in the .ruff.toml file it is mentioned that  "E501",  # line too long (default 88).

Comment (moylop260) [reply stock_lot_production_date/models/stock_lot.py]: @sbidoul Did we change to length 88 or it was a default hidden?

Comment (sbidoul) [reply stock_lot_production_date/models/stock_lot.py]: I don't think we changed anything in that area recently. Probably ruff and black+flake8 have a slightly different interpretation.

File: stock_lot_production_date/models/stock_lot.py
Comment (sbidoul) [inline stock_lot_production_date/models/stock_lot.py]:
```suggestion
        help=("This is the date when the goods with this "
        "lot/serial number have been produced.")
```

Non blocking suggestion: if you add parenthesis like this, ruff will emit something nicer.

Comment (SodexisTeam) [reply stock_lot_production_date/models/stock_lot.py]: @sbidoul Made change as mentioned. Please take a look at it.

**Comment by moylop260** [comment]
@rousseldenis @sebalix 

Any blocking comment?

**Comment by rolandojduartem** [comment]
Hi, what happened with PR? could we merge it? @SodexisTeam @moylop260

File: stock_lot_production_date/__manifest__.py
Comment (rousseldenis) [inline stock_lot_production_date/__manifest__.py]: @SodexisTeam I'd prefer an individual account

Comment (sebalix) [reply stock_lot_production_date/__manifest__.py]: I often forget to put myself in this field, I'm OK to be listed there too :+1:

Comment (sebalix) [reply stock_lot_production_date/__manifest__.py]: I could do that on all versions later myself, it's not a blocker for this migration.

Comment (moylop260) [reply stock_lot_production_date/__manifest__.py]: @sebalix We have a few days waiting for this change to be approved from Roussel

Could you change it, please?

Notice that the maintainers field enables to this github user to merge new PRs

So, it could be better than only one person have this extra access

What do you think?

Comment (rousseldenis) [reply stock_lot_production_date/__manifest__.py]: @moylop260 Do you have contact with @SodexisTeam members ?

We could merge this quickly if the change is done.

Comment (atchuthan) [reply stock_lot_production_date/__manifest__.py]: @rousseldenis @sebalix @moylop260 We have changed the maintainer to individual account as per your suggestion.  Thanks for your patience.

**Review by rousseldenis — APPROVED** [review]

**Comment by rolandojduartem** [comment]
@moylop260 could you review again?

**Review by sebalix — APPROVED** [review]

**Pull request merged**
